eox_core/api/v1/permissions.py

```diff
@@ -15,12 +16,19 @@ def load_permissions():
     to eox-core API.
     """
     if settings.EOX_CORE_LOAD_PERMISSIONS:
-        content_type = ContentType.objects.get_for_model(User)
-        obj, created = Permission.objects.get_or_create(  # pylint: disable=unused-variable
-            codename='can_call_eox_core',
-            name='Can access eox-core API',
-            content_type=content_type,
-        )
+        try:
+            content_type = ContentType.objects.get_for_model(User)
+            obj, created = Permission.objects.get_or_create(  # pylint: disable=unused-variable
+                codename='can_call_eox_core',
+                name='Can access eox-core API',
+                content_type=content_type,
+            )
+        except ProgrammingError as err:
+            # Do not raise exception if it is related to database migrations
```

Comment:
I would add just a more detailed explanation about the issue, indicating that this code runs loading the app before running the migrations, and for that reason we're bypassing the ProgrammingError in this case, to let migrations run smoothly.